modules/ref_ipi-installation-overview.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-prerequisites.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="installation-overview_{context}"]
+= Installation overview
+
+The installation program supports interactive mode, but Red{nbsp}Hat recommends preparing an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts, and the relevant cluster details.
+
+The installation program loads the `install-config.yaml` file, while the administrator generates the manifests and verifies all prerequisites.
+
+The installation program enrolls all nodes in the cluster and starts the bootstrap VM. The installation program starts the metal platform components as `systemd` services, which have the following containers:
+
+* Ironic-dnsmasq: The DHCP server responsible for handing over the IP addresses to the provisioning interface of various nodes on the provisioning network. Ironic-dnsmasq is only enabled when you deploy an {product-title} cluster with a provisioning network.
+* Ironic-httpd: The HTTP server used to ship the images to the nodes.
+* Image-customization
+* Ironic
+* Ironic-inspector (available in {product-title} 4.16 and earlier)
+* Ironic-ramdisk-logs
+* Extract-machine-os
```

Comment:
I think that list items should be parallel. Perhaps, see the Red Hat Technical Writing Style Guide: [For each item in a list, start with either a complete sentence or a sentence fragment](https://stylepedia.net/style/#punc-in-lists).  I think this suggests if one item in the list is defined, they all should be.